scala/build.sbt

```diff
@@ -0,0 +1,16 @@
+scalaVersion := "3.2.2"
+
+enablePlugins(ScalaNativePlugin)
+
+// set to Debug for compilation details (Info is default)
+logLevel := Level.Info
+
+// import to add Scala Native options
+import scala.scalanative.build._
+
+// defaults set with common options shown
+nativeConfig ~= { c =>
+  c.withLTO(LTO.full) // thin
```

Comment:
We should not enable LTO. C/C++/others don't enable LTO. Only whatever the release build enables by default